File: AdvancedCasting/cs/Data/Common/VideoMetaData.cs
//*********************************************************
//
// Copyright (c) Microsoft. All rights reserved.
// This code is licensed under the MIT License (MIT).
// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
//
//*********************************************************

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ScreenCasting.Data.Common
{
    public class VideoMetaData
    {
        public string Id { get; set; }
        public string Title { get; set; }
        public DateTime PubDate { get; set; }        
        public string License { get; set; }
        public Uri WebPageLink { get; set; }
        public Uri VideoLink { get; set; }
        public Uri Thumbnail { get; set; }
    }
}

File: Samples/AdvancedCasting/vb/Data/Common/VideoMetaData.vb
'*********************************************************
'
' Copyright (c) Microsoft. All rights reserved.
' This code is licensed under the MIT License (MIT).
' THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
' ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
' IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
' PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
'
'*********************************************************
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace Global.ScreenCasting.Data.Common

    Public Class VideoMetaData

        Public Property Id As String

        Public Property Title As String

        Public Property PubDate As DateTime

        Public Property License As String

        Public Property WebPageLink As Uri

        Public Property VideoLink As Uri

        Public Property Thumbnail As Uri
    End Class
End Namespace
